html/janus.js

```diff
@@ -2384,26 +2387,35 @@ function Janus(gatewayCallbacks) {
 						pluginHandle.consentDialog(true);
 					}
 					let constraints = Janus.trackConstraints(track), stream = null;
-					if(track.type === 'audio' || track.type === 'video') {
-						// Use getUserMedia: check if we need to group audio and video together
+					if(track.type === 'audio' || (track.type === 'video' || track.type === 'screen') ) {
+						// Set if getUserMedia should be used or if it should be getDisplayMedia
+						let captureUserMedia = track.type !== 'screen';
```

Comment:
Just for the sake of readability, and to be in line with the rest of the code, please wrap the check above in brackets so that it's less ambiguous.